release/sil/sil.bcc-latn.upp_ptwl1/source/sil.bcc-latn.upp_ptwl1.model.ts

```diff
@@ -10,5 +10,36 @@ const source: LexicalModelSource = {
   format: 'trie-1.0',
   wordBreaker: 'default',
   sources: ['wordlist.tsv'],
+  searchTermToKey: function (term) {
+    // Use this pattern to remove common diacritical marks.
+    // See: https://www.compart.com/en/unicode/block/U+0300
+    const COMBINING_DIACRITICAL_MARKS = /[\u0300-\u036f]/g;
+   
+    // Converts to Unicode Normalization form D.
+    // This means that MOST accents and diacritics have been "decomposed" and
+    // are stored as separate characters. We can then remove these separate
+    // characters!
+    //
+    // e.g., � ? A + �
+    let normalizedTerm = term.normalize('NFD');
+   
+    // Now, make it lowercase.
+    //
+    // e.g.,  A + � ? a + �
+    let lowercasedTerm = normalizedTerm.toLowerCase();
```

Comment:
Is there such a thing as lowercasing in Balochi written in Arabic script? If not, then this line can also be removed.